doc/us-en/overview/OpenEdge-design.md

```diff
@@ -432,6 +434,10 @@ def handler(event, context):
 
 _**Tips**: In the native process mode, to run sayhi.py provided in the example of this project, you need to install python2.7 and its packages protobuf3 and grpcio (pip installation can be used, `pip install pyyaml protobuf grpcio`). _
```

Comment:
_**Tips**: In the native process mode, to run sayhi.py provided in the example of this project, you need to install python2.7 and its packages pyyaml, protobuf3 and grpcio (pip installation can be used, `pip install pyyaml protobuf grpcio`). _